commands/new.go

```diff
@@ -0,0 +1,169 @@
+package commands
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/go-morph/morph"
+	"github.com/spf13/cobra"
+
+	. "github.com/dave/jennifer/jen"
+)
+
+const baseDriverPath = "drivers"
+
+func NewCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "new",
+		Short: "Bootstraps new resources",
+	}
+
+	cmd.AddCommand(
+		NewDriverCmd(),
+	)
+
+	return cmd
+}
+
+func NewDriverCmd() *cobra.Command {
+	return &cobra.Command{
+		Use:           "driver",
+		Short:         "Generates necessary file structure for a new driver",
+		Args:          cobra.ExactArgs(1),
+		RunE:          newDriverCmdF,
+		SilenceUsage:  true,
+		SilenceErrors: true,
+	}
+}
+
+func newDriverCmdF(cmd *cobra.Command, args []string) error {
+	driverName := args[0]
+	morph.InfoLogger.Printf("Generating necessary file structure for %q driver\n", driverName)
+
+	if _, err := os.Stat(baseDriverPath + "/" + driverName); !os.IsNotExist(err) {
+		return fmt.Errorf("driver %q already exists, skipping", driverName)
+	}
+
+	return generateNewDriver(driverName)
+}
+
+func generateNewDriver(driverName string) error {
+	morph.InfoLoggerLight.Printf("== CreateDriver: %q: generating driver files\n", driverName)
+	morph.InfoLoggerLight.Println("=================================================")
+
+	f := NewFile(driverName)
```

Comment:
if the only call to a base method of jen is this one, can we change the import to be a named one? (I mean, avoid the `import . "github.com/..."`)

Reply:
unfortunately is not, all the `Id` calls are basically calls to Jen.

Reply:
right, totally missed those. Happy to leave it as is then